[Doc] Add sum reduction for CTCLoss

isdanni opened the pull request:
Summary: 

Fix: #99141

Reference: 
https://github.com/pytorch/pytorch/blob/39b885cbbfc8c115069d49f5a6d27ea622bd05dc/aten/src/ATen/native/LossCTC.cpp#L366-L371

Test Plan: See GitHub Tests.

Differential Revision: D45387774

albanD reviewed (approved):
Thanks!

mikaylagawarecki reviewed (commented):
Thanks @isdanni 

nit: could you also update the documentation for the shape of output per [the comment](https://github.com/pytorch/pytorch/issues/99141#issuecomment-1508312028) on the linked issue please :)

mikaylagawarecki reviewed (approved):
Ah I missed that first scalar! Thank you so much for clarifying the docs! :)

mikaylagawarecki commented:
@pytorchbot rebase

mikaylagawarecki commented:
@pytorchbot merge -f "build failures are unrelated"

isdanni committed:
[Doc] Add sum reduction for CTCLoss (#100235)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/100235

Reference: https://github.com/pytorch/pytorch/blob/39b885cbbfc8c115069d49f5a6d27ea622bd05dc/aten/src/ATen/native/LossCTC.cpp#L366-L371

Test Plan: See GitHub Tests.

Differential Revision: D45387774

fbshipit-source-id: 8166f5ea0383312bce0df2d9ea53aed2edf551b4

isdanni commented:
Thanks for merging this @mikaylagawarecki My other PR had a similar problem. Rebasing and export from Phabricator should do the trick.

mikaylagawarecki commented:
@pytorchbot merge -f "build failure is unrelated"
